app/code/Magento/Inventory/Setup/InstallSchema.php

```diff
@@ -0,0 +1,228 @@
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_DEFAULT => ''
+            ];
+            $table->addColumn(SourceInterface::DESCRIPTION, Table::TYPE_TEXT, 255, $options, 'Description');
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_DEFAULT => ''
+            ];
+            $table->addColumn(SourceInterface::LATITUDE, Table::TYPE_TEXT, 255, $options, 'Latitude');
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_DEFAULT => ''
+            ];
+            $table->addColumn(SourceInterface::LONGITUDE, Table::TYPE_TEXT, 255, $options, 'Longitude');
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_UNSIGNED => true
+            ];
+
+            $table->addColumn(SourceInterface::COUNTRY_ID, Table::TYPE_SMALLINT, null, $options, 'Country Id');
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => true,
+                InstallSchema::OPTION_UNSIGNED => true
+            ];
+
+            $table->addColumn(SourceInterface::REGION_ID, Table::TYPE_SMALLINT, null, $options, 'Region Id');
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_DEFAULT => ''
+            ];
+            $table->addColumn(SourceInterface::REGION, Table::TYPE_TEXT, 255, $options, 'Region');
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_DEFAULT => ''
+            ];
+            $table->addColumn(SourceInterface::CITY, Table::TYPE_TEXT, 255, $options, 'City');
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_DEFAULT => ''
+            ];
+            $table->addColumn(SourceInterface::STREET, Table::TYPE_TEXT, 255, $options, 'Street');
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_DEFAULT => ''
+            ];
+            $table->addColumn(SourceInterface::POSTCODE, Table::TYPE_TEXT, 255, $options, 'Postcode');
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_DEFAULT => ''
+            ];
+            $table->addColumn(SourceInterface::PHONE, Table::TYPE_TEXT, 255, $options, 'Phone');
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => false,
+                InstallSchema::OPTION_DEFAULT => ''
+            ];
+            $table->addColumn(SourceInterface::FAX, Table::TYPE_TEXT, 255, $options, 'Fax');
+
+            $options = [
+                InstallSchema::OPTION_NULLABLE => true,
+                InstallSchema::OPTION_UNSIGNED => true,
+            ];
```

Comment:
it's much easier to read such code when method chaining used , like this:
```php
       )->addColumn(
            'region',
            Table::TYPE_TEXT,
            255,
            ['nullable' => true, 'default' => null],
            'State/Province name'
        )->addColumn(
            'city',
            Table::TYPE_TEXT,
            255,
            ['nullable' => true],
            'City'
        )->addColumn(
            'street',
            Table::TYPE_TEXT,
            null,
            ['nullable' => true],
            'Street Address'
```

please use this format